lib/ddtrace/contrib/faraday/patcher.rb

```diff
@@ -54,15 +53,6 @@ def compatible?
             Gem::Version.new(::Faraday::VERSION) < COMPATIBLE_UNTIL
           end
 
-          def add_pin
```

Comment:
I agree this is probably not necessary anymore, but:
1) It's a breaking change (users might have `Pin.get_from(::Faraday)` in their configuration files;
2) This is branch is meant to port one testing library to another, so if you are changing the test subject, you can't really say the two are equivalent;

Reply:
Hmmm, good point. I really hope they wouldn't considering this pin looks like its managed internally, but it's certainly possible.